### DISCUSSION TITLE
Recently added

### DISCUSSION THREAD

**Discussion by KelvinCrag** [MissingCore/Music#40] [Ideas]
An option to view recently added songs would be nice, or to change the sorting of tracks by date.

**Comment by cyanChill** [comment]
This probably will be added in some point in the future of this app (no estimate time frame) as I plan on adding custom sorting order.

**Comment by cyanChill** [comment]
We've added the ability to change the sort order of tracks on the "Tracks" screen/tab with "alphabetical" and "last modified" options (in ascending or descending order), respecting that order if we play tracks from the "Tracks" screen/tab when not shuffled. If you're using the APK version, you can try it out early with the [`v2.0.0-rc.1`](https://github.com/MissingCore/Music/releases/tag/v2.0.0-rc.1) release.
